server/models/handlers/availability.js

```diff
@@ -1,29 +1,24 @@
 // Gets the time availabilities for a specific worker for a specific school.
-async function getWorkerAvailability(workerId, schoolId, startTime, endTime) {
-    const sampleAvailableTimes = [
-        {
-            workerTimeslotId: 1,
-            startTime: '08:00:00', 
-            endTime: '08:30:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 2,
-            startTime: '08:30:00', 
-            endTime: '09:00:00',
-            date: '2020-10-20'
-        },
-        {
-            workerTimeslotId: 3,
-            startTime: '09:00:00', 
-            endTime: '09:30:00',
-            date: '2020-10-20'
-        }
-    ];
+const _ = require('lodash');
+const availabilityHandler = require('../db/availability');
+//need userTypes or Workerstatus?
+const UserTypes = require('../../constants/userTypes.json');
+
+//
```

Comment:
Hi Nancy,
I am confused about what the `//` represents here. I would recommend removing it if it is unneeded to reduce unnecessarily long files.

Reply:
Hi Amy,
Good catch!
My action: I've removed line 7 in the server/models/handlers/availability.js

Please confirm if this takes care of your suggestion!